### PR TITLE
Improvement/add-linea

### PR DESCRIPTION
**Pull request by sogoiii**
## Describe Changes

I made this better by doing ...

- update networks package that has linea
- modified internal config

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

**Review by alexanderattar — APPROVED** [review]

File: src/utils/contracts.ts [hunk 1]
```diff
@@ -118,4 +118,7 @@ export const LZ_RELAYER_ADDRESSES: {[key: string]: string} = {
   base: '0xb6319cC6c8c27A8F5dAF0dD3DF91EA35C4720dd7'.toLowerCase(),
   zoraTestnetSepolia: '0x55370E0fBB5f5b8dAeD978BA1c075a499eB107B8'.toLowerCase(),
   zora: '0xA658742d33ebd2ce2F0bdFf73515Aa797Fd161D9'.toLowerCase(),
+  lineaGoerli: '0x6EDCE65403992e310A62460808c4b910D972f10f'.toLocaleLowerCase(),
+  lineaSepolia: ''.toLowerCase(),
```
Comment (feliperodriguess) [inline src/utils/contracts.ts:122]: is this expected?

**Review by feliperodriguess — APPROVED** [review]

File: src/utils/contracts.ts [hunk 2]
```diff
@@ -118,4 +118,7 @@ export const LZ_RELAYER_ADDRESSES: {[key: string]: string} = {
   base: '0xb6319cC6c8c27A8F5dAF0dD3DF91EA35C4720dd7'.toLowerCase(),
   zoraTestnetSepolia: '0x55370E0fBB5f5b8dAeD978BA1c075a499eB107B8'.toLowerCase(),
   zora: '0xA658742d33ebd2ce2F0bdFf73515Aa797Fd161D9'.toLowerCase(),
+  lineaGoerli: '0x6EDCE65403992e310A62460808c4b910D972f10f'.toLocaleLowerCase(),
```
Comment (feliperodriguess) [inline src/utils/contracts.ts]: Your using `toLocaleLowerCase()` i think this was an oversight, change to `toLowerCase()`

File: src/utils/contracts.ts (hunk 1 above)
Comment (sogoiii) [reply src/utils/contracts.ts:122]: @feliperodriguess yea. LZ has not deployed to that new network yet.

File: src/utils/contracts.ts (hunk 2 above)
Comment (sogoiii) [reply src/utils/contracts.ts]: @feliperodriguess ok, fixed.

**Pull request merged**